gensim/models/word2vec.py

```diff
@@ -776,6 +795,9 @@ def job_producer():
                             progress = 1.0 * pushed_words / total_words
                         next_alpha = self.alpha - (self.alpha - self.min_alpha) * progress
                         next_alpha = max(self.min_alpha, next_alpha)
+                        # raise warning in case alpha rises again
+                        if next_alpha > current_alpha:
+                            warnings.warn("'alpha' risen again after descending.")
```

Comment:
Sorry for that I misunderstood the problem. So once alpha rises again, would it be better to keep track of the new alpha or keep the lowest value yet reached? I'm just keeping a variable called `min_alpha_yet_reached` which is initialized as `alpha`. I keep checking in every training cycle if this is greater than `alpha` and set it to `alpha` if so. Whereas if it is lesser, I was thinking a warning should be raised and the `min_alpha_yet_reached` should be set to the raised `alpha` so as to eliminate the noise in the log. Is this approach fine?